src/sentry/static/sentry/app/views/settings/components/forms/form.jsx

```diff
@@ -152,7 +153,7 @@ export default class Form extends React.Component {
         <div>{children}</div>
 
         {shouldShowFooter && (
-          <div className={footerClass} style={{marginTop: 25, ...footerStyle}}>
+          <StyledFooter className={footerClass} style={footerStyle}>
```

Comment:
This might break some forms that are still setting the modal-footer class. https://github.com/getsentry/sentry/blob/ae5e7d7769a7cbba326db135c7f3a6456ba27146/src/sentry/static/sentry/app/components/group/externalIssueActions.jsx#L169